pkg/metrics/registry.go

```diff
@@ -0,0 +1,170 @@
+// Package `metrics` provides some tools useful for gathering and
+// exposing system metrics for external monitoring tools.
+//
+// Currently, this package is intended to use with Prometheus but
+// can be easily extended if needed. Also, not all Prometheus metric
+// types are implemented. The main motivation of creating a custom
+// package was a need to avoid usage of external unaudited dependencies.
```

Comment:
```
// The main motivation of creating a custom
// package was a need to avoid usage of external unaudited dependencies.
```
Let's :fire: it, it's not important in this context.